windows-gaming-driver/src/driver/controller.rs

```diff
@@ -37,10 +38,11 @@ enum State {
     Resuming,
 }
 
-#[derive(PartialEq, Eq, Clone, Copy)]
+#[derive(Clone)]
```

Comment:
Nope, the matches are not affected. The larger problem here was `PartialEq` being gone which meant that I had to rewrite every `if self.io_state == foo` to `let foo = self.io_state`. Anyway, this is definitely the cleaner solution. I used the `Option<Sender>` approach before, but this is better.